PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchViewModel.swift

```diff
@@ -54,6 +54,8 @@ class SearchViewModel: ObservableObject {
         }
     }
 
+    @Published var searchText = ""
```

Comment:
we can add a `didSet` here so that we know that whenever this is updated, then we want to call `updateSearchResults `

could we rename this to something similar to `recentSearchTerm` so we have context that this is for recent search specifically?